pkg/asset/installconfig/gcp/session.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	authEnvs            = []string{"GOOGLE_CREDENTIALS", "GOOGLE_CLOUD_KEYFILE_JSON", "GCLOUD_KEYFILE_JSON"}
+	authEnvs            = []string{"GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_CREDENTIALS", "GOOGLE_CLOUD_KEYFILE_JSON", "GCLOUD_KEYFILE_JSON"}
```

Comment:
nit: This should be added to the comment in GetSession as well so users know the order that environment variables are selected.